doc/changelog.asciidoc

```diff
@@ -30,6 +30,10 @@ Removed
 
 - Support for macOS 11 Big Sur is dropped. Binaries are now built on macOS 12
   Monterey and are unlikely to still run on older macOS versions.
+- Failed end2end tests will now save screenshots of the browser window when
```

Comment:
@The-Compiler should this be under the "Removed" section? It seems like it might be misplaced, looking at the changes in the PR.

Reply:
That indeed seems like a mistake, well spotted! Pushed a commit to fix it.